[2022-06-24 10:05:00.895] [remoteagent] [info] Extension host agent started.
[2022-06-24 10:05:01.417] [remoteagent] [info] [127.0.0.1][3a576515][ManagementConnection] New connection established.
[2022-06-24 10:05:01.424] [remoteagent] [info] [127.0.0.1][00f983dd][ExtensionHostConnection] New connection established.
[2022-06-24 10:05:01.433] [remoteagent] [info] [127.0.0.1][00f983dd][ExtensionHostConnection] <17294> Launched Extension Host Process.
[2022-06-24 10:05:12.357] [remoteagent] [info] [127.0.0.1][3a576515][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-06-24 10:05:18.933] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] New connection established.
[2022-06-24 10:05:18.951] [remoteagent] [info] [127.0.0.1][6638f27d][ExtensionHostConnection] New connection established.
[2022-06-24 10:05:18.956] [remoteagent] [info] [127.0.0.1][6638f27d][ExtensionHostConnection] <17500> Launched Extension Host Process.
[2022-06-24 10:05:21.632] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 10:10:19.143] [remoteagent] [info] [127.0.0.1][00f983dd][ExtensionHostConnection] <17294> Extension Host Process exited with code: 0, signal: null.
[2022-06-24 10:26:29.462] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-24 10:26:29.463] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-24 10:29:03.727] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-24 10:29:03.730] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] The client has reconnected.
[2022-06-24 10:29:04.778] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 10:29:04.779] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 10:29:04.780] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 10:29:05.698] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 10:29:28.769] [remoteagent] [info] [127.0.0.1][6638f27d][ExtensionHostConnection] The client has reconnected.
[2022-06-24 11:10:05.820] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] The client has reconnected.
[2022-06-24 11:10:06.885] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 11:10:06.887] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 11:10:06.893] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 11:10:08.086] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 11:10:31.360] [remoteagent] [info] [127.0.0.1][6638f27d][ExtensionHostConnection] The client has reconnected.
[2022-06-24 12:12:30.767] [remoteagent] [info] [127.0.0.1][e2535f9c][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-06-24 12:14:11.735] [remoteagent] [info] [127.0.0.1][13dfcfa1][ManagementConnection] New connection established.
[2022-06-24 12:14:11.757] [remoteagent] [info] [127.0.0.1][8e9b38e4][ExtensionHostConnection] New connection established.
[2022-06-24 12:14:11.771] [remoteagent] [info] [127.0.0.1][8e9b38e4][ExtensionHostConnection] <11970> Launched Extension Host Process.
[2022-06-24 12:19:11.960] [remoteagent] [info] [127.0.0.1][6638f27d][ExtensionHostConnection] <17500> Extension Host Process exited with code: 0, signal: null.
[2022-06-24 12:39:52.476] [remoteagent] [info] [127.0.0.1][13dfcfa1][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-24 12:39:52.482] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-24 12:43:40.015] [remoteagent] [info] [127.0.0.1][13dfcfa1][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-24 12:43:40.016] [remoteagent] [info] [127.0.0.1][3c7bd307][ManagementConnection] New connection established.
[2022-06-24 12:43:40.022] [remoteagent] [info] [127.0.0.1][bc83d845][ExtensionHostConnection] New connection established.
[2022-06-24 12:43:40.033] [remoteagent] [info] [127.0.0.1][bc83d845][ExtensionHostConnection] <19595> Launched Extension Host Process.
[2022-06-24 12:48:40.196] [remoteagent] [info] [127.0.0.1][13dfcfa1][ManagementConnection] The reconnection short grace time of 5m has expired, so the connection will be disposed.
[2022-06-24 12:48:40.354] [remoteagent] [info] [127.0.0.1][8e9b38e4][ExtensionHostConnection] <11970> Extension Host Process exited with code: 0, signal: null.
[2022-06-24 13:41:27.699] [remoteagent] [info] [127.0.0.1][3c7bd307][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[2022-06-24 13:41:27.708] [remoteagent] [error] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/server/node/server.main.js:143:7923)
    at process.emit (node:events:390:28)
[2022-06-24 14:26:09.036] [remoteagent] [info] [127.0.0.1][3c7bd307][ManagementConnection] Another client has connected, will shorten the wait for reconnection 5m before disposing...
[2022-06-24 14:26:09.039] [remoteagent] [info] [127.0.0.1][3c7bd307][ManagementConnection] The client has reconnected.
[2022-06-24 14:26:09.131] [remoteagent] [info] [127.0.0.1][bc83d845][ExtensionHostConnection] The client has reconnected.
[2022-06-24 14:26:10.127] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateIcon (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1733)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 14:26:10.129] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 14:26:10.131] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 14:26:11.317] [remoteagent] [error] ["Error: Could not find pty on pty host","    at w._throwIfNoPty (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:5333)","    at w.updateTitle (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:20:1673)","    at Object.call (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:13:8386)","    at h.onPromise (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4891)","    at h.onRawMessage (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:4274)","    at /home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:12:3560","    at E.invoke (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:145)","    at W.deliver (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:2271)","    at c.fire (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:8:1849)","    at process.U (/home/peci2022/.vscode-server/bin/30d9c6cd9483b2cc586687151bcbcd635f373630/out/vs/platform/terminal/node/ptyHostMain.js:6:35698)","    at process.emit (node:events:390:28)","    at emit (node:internal/child_process:917:12)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-06-24 14:26:11.630] [remoteagent] [info] [127.0.0.1][3c7bd307][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-06-24 17:26:23.393] [remoteagent] [info] [127.0.0.1][bc83d845][ExtensionHostConnection] <19595> Extension Host Process exited with code: 0, signal: null.
[2022-06-24 17:26:23.394] [remoteagent] [info] Last EH closed, waiting before shutting down
[2022-06-24 17:31:23.395] [remoteagent] [info] Last EH closed, shutting down
